sphinx/transforms/post_transforms/code.py

```diff
@@ -112,11 +112,19 @@ def apply(self, **kwargs):
 
         for node in self.document.traverse(nodes.literal_block):
             if self.is_pyconsole(node):
-                source = node.rawsource
-                source = doctest.blankline_re.sub('', source)
-                source = doctest.doctestopt_re.sub('', source)
-                node.rawsource = source
-                node[:] = [nodes.Text(source)]
+                self.strip_doctest_flags(node)
+
+        for node in self.document.traverse(nodes.doctest_block):
+            self.strip_doctest_flags(node)
+
+    @staticmethod
+    def strip_doctest_flags(node):
+        # type: (Union[nodes.literal_block, nodes.doctest_block]) -> None
```

Comment:
mypy warns this line:
```
sphinx/transforms/post_transforms/code.py:120: error: Name 'Union' is not defined
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/553354669

Could you import `Union` please?